src/types/enterprise.ts

```diff
@@ -2,7 +2,12 @@ export type EnterpriseCobranding = {
   text: string
 }
 
+export type EnterpriseLogoCobranding = {
+  src: string
+}
+
```

Comment:
Just wondering, why does this logo cobranding need to be a separate config to the existing cobranding one?
Was there issues if it was structured this way, then let the display priority be based on if `src` is defined else use `text`?
```
export type EnterpriseCobranding = {
  text: string
  src: string
}
```

Reply:
I made it a separate object mostly for the validation of the feature in the jwt. `cobrand` and `logoCobrand` are two entirely separate features in that aspect, with `cobrand` meaning text cobranding and `logoCobrand` meaning image cobranding. I would've updated the `EnterpriseCobranding` object to be `EnterpriseTextCobranding` but didn't want to risk any breaking changes for integrators.